gbpservice/contrib/nfp/tools/image_builder/disk_image_create.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+
+import commands
+import datetime
+import json
+import os
+import subprocess
+import sys
+
+conf = {}
```

Comment:
Lets us have a class in this file to contain all the methods and maintain the conf and any other variables as class member variables. Lets do this as part of fixing devstack.